### PR TITLE
batch upload modified coords (fix #7028)

### LINKED ISSUE #7028: Batch upload modified cache coords to the website
Feature request:

c:geo has a nice feature to upload corrected coordinates to geocaching.com website. I frequently have the wish to do this for multiple caches at once (eg: after solving a complete trail's puzzles on my mobile device or after importing a gpx file into c:geo). This function would then upload the corrected coordinates cache by cache.

A good place for this function would be (IMHO): stored - dot menu - manage caches and then maybe "upload coordinates".

This could be implemented for personal notes as well. (This is the place where I put information on how I have solved the puzzle.)

### PR DESCRIPTION
**Pull request by moving-bits**
add menu entry on cache list activity to batch upload all modified coords (if connector allows)

### CODE REVIEW

File: main/src/cgeo/geocaching/export/BatchUploadModifiedCoordinates.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            if (activity != null) {
+                final Context nonNullActivity = activity;
+                if (result) {
+                    ActivityMixin.showToast(activity, uploadFailed == 0
+                            ? nonNullActivity.getString(R.string.export_modifiedcoords_success)
+                            : nonNullActivity.getString(R.string.export_modifiedcoords_result, uploadFailed, uploadOk));
+                } else {
+                    ActivityMixin.showToast(activity, nonNullActivity.getString(R.string.export_modifiedcoords_error));
+                }
+                Log.e("upload finished: ok=" + uploadOk + " / failed=" + uploadFailed + " / total=" + uploadProgress);
```
[the remark this answers is not in the document]
Comment (moving-bits) [reply main/src/cgeo/geocaching/export/BatchUploadModifiedCoordinates.java]: Good catch, thanks. I used it to get the message through my filtering on errors only. I will either change or remove it.

[the remark this answers is not in the document]
Comment (Lineflyer) [reply main/src/cgeo/geocaching/export/BatchUploadModifiedCoordinates.java]: In general some meaningful `log.d` might be helpful for me fieldtesting later on....e.g. in error cases or especially for detail outcome of batch operations.
Don't know, what exactly is the best practice for this in java coding.

[the remark this answers is not in the document]
Comment (moving-bits) [reply main/src/cgeo/geocaching/export/BatchUploadModifiedCoordinates.java]: fixed, squashed and force-pushed

**Comment by moving-bits** [comment]
retest this please

**Comment by Lineflyer** [comment]
@moving-bits Can you double confirm you consider this change ready to merge?`We just copied `master` to `release`so its the best time for new feature merging onto `master`.

**Comment by Lineflyer** [comment]
@moving-bits See my last comment?!

**Pull request merged**